content/docs/rfcs/67/README.md

```diff
@@ -0,0 +1,200 @@
+
+### Waku Node:
+
+Software that is configured with a set of Waku protocols.
+A Status client comprises of a Waku node that can be a relay node or
+a non-relay node.
+
+The Waku network is a group of Waku nodes used to create an open access messaging network, 
+as described in [64/WAKU2-NETWORK](/spec/64).
+
+### Light Client:
+
+A Status client that operates within resource constrained environments is a node configured as light client.
+Light clients do not run `RELAY`.
+Instead, using a Waku service protocol, nodes can request services from other `RELAY` nodes.
+
+# Protocol Usage
+
+The following is a list of Waku Protocols used by the Status application.
+
+## 1. `RELAY`
+
+The `RELAY` MUST not be used by Status light clients.
+The `RELAY` is used to broadcast messages from a Status client nodes to peers in the [64/WAKU2-NETWORK](/spec/64).
+All Status messages are transformed into [14/WAKU2-MESSAGE](/spec/14), which are sent over the wire.
+
+All Status message types are described in [62/STATUS-PAYLOAD](/spec/62).
+Status Clients MUST transform the following object into a `MESSAGE` as described below -
+
+```go
+
+type StatusMessage struct {
+    SymKey[]     []byte // [optional] The symmetric key used to encrypt the message
+    PublicKey    []byte // [optional] The public key to use for asymmetric encryption
+    Sig          string // [optional] The private key used to sign the message
+    PubsubTopic  string // The Pubsub topic to publish the message to     
+    ContentTopic string // The Content topic to publish the message to
+    Payload      []byte // A serialized representation of a Status message to be sent
+    Padding      []byte // Padding that must be applied to the Payload
+    TargetPeer   string // [optional] The target recipient of the message
+    Ephemeral    bool   // If the message is not to be stored, this is set to `true` 
+}
+
+```
+
+1. A user MUST only provide either a Symmetric key OR an Asymmetric keypair to encrypt the message.
+If both are received, the implementation MUST throw an error.
+2. `WakuMessage.Payload` MUST be set to `StatusMessage.Payload` 
+3. `WakuMessage.Key` MUST be set to `StatusMessage.SymKey`
+4. `WakuMessage.Version` MUST be set to `1`
+5. `WakuMessage.Ephemeral` MUST be set to `StatusMessage.Ephemeral`
+6. `WakuMessage.ContentTopic` MUST be set to `StatusMessage.ContentTopic`
+7. `WakuMessage.Timestamp` MUST be set to the current Unix epoch timestamp (in nanosecond precision)
+
+## 2. `STORE`
+
+> Note: This protocol MUST remain optional according to the user's preferences,
+it MAY be enabled on Light clients as well.
+
+Messages received via [11/WAKU2-RELAY](/spec/11), are stored in a database.
+When Waku node running this protocol is service node, 
+it MUST provide the complete list of network messages.
+Status clients COULD request historical messages from this service node.
```

Comment:
`COULD` is not an RFC keyword: https://www.ietf.org/rfc/rfc2119.txt. Use `MAY` or `SHOULD`.